faq.html

```diff
@@ -0,0 +1,155 @@
+                    Alternatively, drag and drop games to <code>Vita3K.exe</code>.<br /><br />
+                    Windows/Linux/macOS:<br />
+                    Games can be installed via the emulator directly through the <code>File > Install .pkg</code> or <code>File > Install .zip .vpk</code>.<br />
+                    Supply them as a command-line argument to the executable (the latter is necessary for Linux/Mac OS).
+                    Alternatively, install them manually by unzipping the <code>.vpk</code>  to Vita3K's home directory (see below).
+                </h6>
+            </div>
+            <h3>Where are games installed?</h3>
+            <div class="my-5">
+                <h6>
+                    They are installed in the Vita3K home directory, in <code><Vita3K_home_dir>/ux0/app/<Title_ID></code>.
+                    Defaults are:<br />
+                    Windows: <code>C:/Users/<username>/AppData/Roaming/Vita3K/Vita3k</code><br />
+                    Linux: <code>~/.local/share/Vita3K/Vita3K</code><br />
+                    You can change it by editing <code>config.yml</code>'s "<code>pref-path</code>" entry.
+                </h6>
+            </div>
+            <h3>Can I play multiplayer games online with real consoles or other Vita3K users?</h3>
+            <div class="my-5">
+                <h6>
+                    Playing commercial games online with real PlayStation Vita systems would require the user to connect to PlayStation Network which isn't very feasible due to obvious technical and legal limitations.
+                    That said, networking is partly implemented and you can play some <strong>homebrew</strong> multiplayer games like <a href="https://github.com/Rinnegatamante/vitaQuake">vitaQuake</a>just fine.
+                </h6>
+            </div>
+            <h3>Where to download firmwares?</h3>
+            <div class="my-5">
+                <h6>
+                    Go to <code>Help</code> -> <code>Welcome</code> -><code>Download Firmware</code>. This will decrypt the modules only.
+                    If you also need all firmware files, like fonts, go to <code>Help</code> -> <code>Welcome</code> -><code>Download Firmware Font Package</code>.
+                </h6>
+            </div>
+            <h3>How to play Vita3K on Steam Deck?</h3>
+            <div class="my-5">
+                <h6>
+                    Using Vita3K on Steam Deck:<br /><br />
+                    0. Switch to desktop mode (hold power button until you get options menu, select Switch to Desktop)<br />
+                    1. Open Konsole<br />
+                    2. run mkdir Vita3K && cd Vita3K && wget <a href="//github.com/Vita3K/Vita3K/releases/download/continuous/ubuntu-latest.zip">github.com/Vita3K/Vita3K/releases/download/continuous/ubuntu-latest.zip</a><br />
+                    3. rununzip ubuntu-latest.zip<br />
```

Comment:
space between run and unzip please

Reply:
would also be good to wrap the commands in <code> tags to show that they're terminal commands